test/libsolidity/semanticTests/builtins/balance_with_balance.sol

```diff
@@ -0,0 +1,8 @@
+contract ClientReceipt {
+    constructor() payable {}
+}
+// ====
+// compileViaYul: also
+// ----
+// constructor(), 1000 wei ->
```

Comment:
Can you also add another test with `1 ether` or something like that?

Reply:
Added in `balance_with_balance2.sol`